PWGDQ/TableProducer/tableMaker.cxx

```diff
@@ -1326,6 +1339,13 @@ struct TableMaker {
     }
   }
 
+  // Produce barrel + muon tables for the eletron-muon analysis, without PIDTOF----------------------------------------------------------------------
+  void processFullForElectronMuon(MyEvents::iterator const& collision, aod::BCsWithTimestamps const& bcs,
+                                  soa::Filtered<MyBarrelTracksForElectronMuon> const& tracksBarrel, soa::Filtered<MyMuons> const& tracksMuon)
+  {
+    fullSkimming<gkEventFillMap, gkTrackFillMapForElectronMuon, gkMuonFillMap>(collision, bcs, tracksBarrel, tracksMuon, nullptr, nullptr);
+  }
+
```

Comment:
Hi Maolin, I am not sure I understand the need of this process function. Could you explain? One can get already the muons and electrons in the data model with the other "Full" process functions.

Reply:
Hi Ionut, this is our current solution for the memory consumption problem on the hyperloop. With other full processes, the memory consumption is always larger than the limit 3.6 G. So I add this new "slight" process with only TPCPID, and remove all the unnecessary PID dependencies.

Reply:
Hi Maolin, ok i will approve this, but we have to look more into this. Not having TOF in the skims can be problematic if the TPC pid is not enough.